src/Core/Content/Test/Cms/SalesChannel/SalesChannelCmsPageRepositoryTest.php

```diff
@@ -147,6 +147,21 @@ private function createPage(): string
                         ],
                     ],
                 ],
+                [
+                    'id' => Uuid::randomHex(),
+                    'type' => 'default',
+                    'position' => 1,
```

Comment:
you trying to test the position sorting. Please set this position to 2 and add an assertion in the test for:

`$page->getSections()->first()->getPosition === 1`